opba-db/src/test/java/de/adorsys/opba/db/SqlConvertTest.java

```diff
@@ -0,0 +1,151 @@
+package de.adorsys.opba.db;
+
+import com.google.common.io.FileWriteMode;
+import com.google.common.io.Files;
+import com.google.common.io.Resources;
+import de.adorsys.opba.db.config.TestConfig;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.util.List;
+
+import static de.adorsys.opba.db.SqlConvertTest.ENABLE_SQL_CONVERSION;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+@SpringBootTest(classes = TestConfig.class)
```

Comment:
Add JavaDoc that describes that it is not a test but SQL generator for bank actions

Reply:
java doc was added